app/components/footer.hbs

```diff
@@ -8,10 +8,12 @@
     <span local-class="sep">|</span>
     <a href='https://www.rust-lang.org/policies/security'>Report a security issue</a>
     <span local-class="sep">|</span>
+    <a href='https://github.com/rust-lang/crates.io/issues/new/choose'>Report an issue</a>
```

Comment:
I couldn't attend the meeting so I have no context, but I think it's clearer to say "Report a general issue", considering we have "Report a security issue". And I'd prefer to place the security link next to the general link.

Reply:
Make sense. Addressed. Thanks for your review!